Packs/Mattermost/Integrations/MattermostV2/README.md

```diff
@@ -651,6 +707,223 @@ No permissions channel.
+
+#### Required Permissions
+
+Must have manage_system permission.
+
+#### Base Command
+
+`mattermost-list-group-members`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| page | The page number to retrieve. Default value is 0. | Optional | 
+| page_size | The size of the page to retrieve. Default value is 50. | Optional | 
+| limit | How many results to retrieve. Will override the page and page_size arguments if given. | Optional | 
+| group_id | The group id of the user group to list members for. Use the command 'mattermost-list-groups' to fetch the group ID. | Required | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Mattermost.Groups.id | String | The ID of the user group. | 
+| Mattermost.Groups.total_member_count | Unknown | The total count of members in the user group. | 
+| Mattermost.Groups.members | Unknown | Detailed list of the user group members. | 
+
+#### Command example
+
+```!mattermost-list-group-members group_id=group_id```
+
+#### Human Readable Output
+
+>### Group moi9ygz8qby1pr1xgkcfuqww9r members (2/2):
+
+>|username|email|id|
+>|---|---|---|
+>| username1 | admin@admin.com | 8a6t7whumbdbxrawretujh6rre |
+>| username2 | admin@ddev.com | o9hpcwz73fdwxe9adue8jxo16o |
+
+### mattermost-add-group-member
+
+***
+Add group member(s). Groups are available on Enterprise and Professional plans. Minimum server version: 6.3
+
+#### Required Permissions
+
+Must have custom_group_manage_members permission for the given group.
+
+#### Base Command
+
+`mattermost-add-group-member`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| group_id | The group id of the user group to list members for. Use the command 'mattermost-list-groups' to fetch the group ID. | Required | 
+| user_ids | The ID(s) of the user(s) to add. Use the command 'mattermost-list-users' to fetch the user ID(s). | Required | 
+
+#### Context Output
+
+There is no context output for this command.
+
+#### Command example
+
+```!mattermost-add-group-member group_id=group_id user_ids=user_ids```
+
+#### Human Readable Output
+
+>The member username was added to the group successfully, with group ID: moi9ygz8qby1pr1xgkcfuqww9r
+
+### mattermost-remove-group-member
+
+***
+Remove group member(s). Groups are available on Enterprise and Professional plans. Minimum server version: 6.3
+
+#### Required Permissions
+
+Must have custom_group_manage_members permission for the given group.
+
+#### Base Command
+
+`mattermost-remove-group-member`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| group_id | The group id of the user group to list members for. Use the command 'mattermost-list-groups' to fetch the group ID. | Required | 
+| user_ids | The ID(s) of the user(s) to remove. Use the command 'mattermost-list-users' to fetch the user ID(s). | Required | 
```

Comment:
Is this a comma-separated list?
If so, the description should be: A comma-separated list of IDs of the users to remove......